phoenix-core/src/main/java/org/apache/phoenix/util/IndexUtil.java

```diff
@@ -39,11 +37,12 @@
 import java.util.List;
 import java.util.ListIterator;
 import java.util.Map;
-import java.util.NavigableSet;
 import java.util.concurrent.TimeUnit;
 
 import com.google.common.cache.Cache;
 import com.google.common.cache.CacheBuilder;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
```

Comment:
nit: new import statement but no use in this file?

Reply:
+1